index.yaml

```diff
@@ -3233,6 +3277,21 @@ entries:
     urls:
     - assets/longhorn-crd/longhorn-crd-102.3.0+up1.5.1.tgz
     version: 102.3.0+up1.5.1
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: longhorn-system
+      catalog.cattle.io/release-name: longhorn-crd
+    apiVersion: v1
+    appVersion: v1.4.3
+    created: "2023-07-17T12:05:59.942611+08:00"
+    description: Installs the CRDs for longhorn.
+    digest: 00986f4afd2def8213d22c03e7662eb08999ea8769be599de6772e79c8865f4e
+    name: longhorn-crd
+    type: application
+    urls:
+    - assets/longhorn-crd/longhorn-crd-102.2.2+up1.4.3.tgz
+    version: 102.2.2+up1.4.3
```

Comment:
This is a PR for 102.3.0+up1.5.1 but you are adding for 102.2.2+up1.4.3. 

Can you maybe rebase this PR ?

Reply:
Outdated Comment.